scripts/lint.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+die() {
+	echo "ERR: $*"
+	exit 1
+}
+
+# Check that no .pb.go files were added.
```

Comment:
probably don't need since we don't generate go from protos?

Reply:
fair enough, let me axe it